app/assets/javascripts/application.js

```diff
@@ -6,11 +6,16 @@
 //= require length_counter
 //= require markdown_preview
 //= require toggle_display_with_checked_input
+//= require modules/paste-html-to-govspeak.js
 
 jQuery(function ($) {
   $('.js-length-counter').each(function () {
     new GOVUK.LengthCounter({ $el: $(this) }) // eslint-disable-line no-new
   })
 
   $('.reorderable-document-list').sortable()
+
+  $('.js-paste-html-to-govspeak').each(function () {
+    new GOVUK.Modules.PasteHtmlToGovspeak({ $el: $(this) }) // eslint-disable-line no-new
+  })
```

Comment:
I'm not sure why this is needed as it doesn't look like we make use of the class `.js-paste-html-to-govspeak` anywhere? Similarly it doesn't look like we set `GOVUK.Modules.PasteHtmlToGovspeak`